Split/Matchers/Semver/InListSemverMatcher.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  InListSemverMatcher.swift
+//  Split
+//
+//  Copyright © 2024 Split. All rights reserved.
+//
+
+import Foundation
+
+class InListSemverMatcher: BaseMatcher, MatcherProtocol {
+
+    var targetList: Set<Semver> = Set()
+
+    init(data: [String]?,
+         negate: Bool? = nil,
+         attribute: String? = nil,
+         type: MatcherType? = nil) {
+        super.init(negate: negate, attribute: attribute, type: type)
+        self.targetList = Set<Semver>(data?.compactMap { item in
+            Semver.build(version: item)
```

Comment:
what happen if `item` is a wrong semver version? are you adding a `null` item into the list?

Reply:
No, `compactMap` will filter out the `nil` values